misc/python/materialize/checks/all_checks/statement_logging.py

```diff
@@ -58,14 +58,14 @@ def validate(self) -> Testdrive:
             return Testdrive(
                 dedent(
                     """
-                    > SET SESSION enable_session_rbac_checks TO false
+                    > SET LOCAL enable_session_rbac_checks TO false
```

Comment:
I don't think local is correct, it will only be active for a single transaction, session sounds better.

Reply:
`SESSION` was not working (https://buildkite.com/materialize/nightly/builds/9749) so I wanted to give it a try. But yeah, it probably won't work either.

Reply:
Maybe the way it is implemented is that any session can *enable* session-wide rbac, but you can't disable rbac in a session when it's enabled system-wide. That would make sense to me at least. If it doesn't work, we can also check with Joe.

Reply:
I will reach out to Joe again.

Reply:
Yeah, that's the case (you cannot disable it for a session) and that makes sense to me.